.travis.yml

```diff
@@ -59,6 +27,7 @@ before_install:
       sudo apt-get update
       sudo apt-get install -y libc6:i386 libncurses5:i386 libstdc++6:i386
       sudo apt-get install -y lcov
+      sudo apt-get install -y libmatio libmatio-dev
```

Comment:
you just need libmatio-dev

Reply:
libmatio-dev only provides .a + includes.

Reply:
.so belongs to libmatio2